cep-20.2.md

```diff
@@ -0,0 +1,831 @@
+schema_version: 1 # integer
+```
+
+To benefit from autocompletion, and other LSP features in editors, we can add a schema URL to the recipe.
+
+```yaml
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+```
+
+## Context section
+
+```yaml
+# The context dictionary defines arbitrary key-value pairs for Jinja interpolation
+# and replaces the {% set ... %} commands commonly used in recipes
+context:
+  variable: test
+  # note that we can reference previous values, that means that they are rendered in order
+  other_variable: test_${{ variable }}
+```
+
+## Package section
+
+```yaml
+package:
+  # The name of the package
+  name: string
+  # The version of the package, following the conda version spec
+  # Note that versions are always strings (1.23 is _not_ a valid version since it is a `float`. Needs to be quoted.)
+  version: string
+```
+
+## Build section
+
+```yaml
+build:
+  # the build number
+  number: Option<integer>, defaults to 0
+
+  # the build string. This is usually ommited (can use `${{ hash }}` variable here)
+  string: Option<string>, defaults to a build string made from "package hash & build number"
+
+  # A list of conditions under which to skip the build of this package (they are joined by `or`)
+  skip: [list of expressions]
+
+  # wether the package is a noarch package, and if yes, wether it is "generic" or "python"
+  noarch: Option<"generic" | "python">
+
+  # script can be a single string or a list of strings
+  # if script is a single string and ends with `.sh` or `.bat`, then we interpret it as a file
+  script: str | [str] | Script
+
+  # would love to get rid of this:
+  # merge the build and host environments (used in many R packages on Windows)
+  # was `merge_build_host`
+  merge_build_and_host_envs: bool (defaults to false)
+
+  # include files even if they are already in the environment
+  # as part of some other host dependency
+  always_include_files: [path]
+
+  variant:
+    # Keys to forcibly use for the variant computation (even if they are not in the dependencies)
+    use_keys: [string]
+
+    # Keys to forcibly ignore for the variant computation (even if they are in the dependencies)
+    ignore_keys: [string]
+
+    # used to prefer this variant less
+    # note: was `track_features`
+    down_prioritize_variant: integer (negative, defaults to 0)
+
+  # settings concerning only Python
+  python:
+    # PythonEntryPoint: `bsdiff4 = bsdiff4.cli:main_bsdiff4`
+    entry_points: [PythonEntryPoint]
+
+    # Specifies if python.app should be used as the entrypoint on macOS
+    # was `osx_is_app`
+    use_python_app_entrypoint: bool (defaults to false)  # macOS only!
+
+    # used on conda-forge, still needed?
+    preserve_egg_dir: bool (default false)
+
+    # skip compiling pyc for some files (was `skip_compile_pyc`)
+    skip_pyc_compilation: [glob]
+
+    # This is only used in the pip feedstock.
+    disable_pip: bool (defaults to false)
+
```

Comment:
Let's outright remove this.
This is more or less just a redundant thing for `conda-build` to do the same as `add_pip_as_python_dependency: false`.
We should not carry these one-off settings over to the new format.
conda-forge already builds with `add_pip_as_python_dependency: false` generally -- if Anaconda or whoever else builds `pip` as a Conda package doesn't do it, they just should.